.gitignore

```diff
@@ -43,3 +43,7 @@ jspm_packages
 
 # We need to include this folders, because they are mocks for integration tests
 !tests/**/node_modules
+
+.vscode
+.idea
+**/node_modules
```

Comment:
It's a bad idea. We need that folder in each integration test case, because without it, you'll always test version from root `node_modules`.